### DISCUSSION TITLE
How to get the elementor rendered full post content HTML?

### DISCUSSION THREAD

**Discussion by vaakash** [elementor/elementor#19061] [Developers Community]
When I use `get_the_content()` it returns the simple HTML. But I want to get the final HTML that elementor renders on the post. I even tried applying `the_content` filter, but the result is just the basic HTML.

There is a question in StackOverflow for the same.
https://stackoverflow.com/questions/71538470/is-it-possible-to-get-elementor-style-of-post-content-in-post-post-content

Can someone please help me with this?

**Comment by m1ga** [comment]
does this work for you https://stackoverflow.com/a/72838847/5193915

**Reply by vaakash** [reply]
Thanks this works. Is there any reason why there is no single API/function to fetch the rendered content of a elementor post? I wrapper like `elementor_get_content( post_id )` would do good.
